src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -71,6 +74,7 @@ public int Main(string[] args)
                     command.FeedOptions = feedOptions;
                     command.Lock = optLock.HasValue();
                     command.Unlock = optUnlock.HasValue();
+                    command.RestoreInParallel = restoreInParallel.HasValue();
```

Comment:
`RestoreCommand.RestoreInParallel`? Should we get rid of unnecessary redundancy and make it `RestoreCommand.Parallel`?